UnityProject/Assets/Scripts/Core/SecureStuff/HubValidation.cs

```diff
@@ -104,21 +127,36 @@ private static void SaveCashedURLConfiguration(URLData URLData)
 
 		}
 
-		private static void AddTrustedHost(string Host, bool IsAPI)
+		private static void AddTrustedHost(Uri URL, bool IsAPI)
 		{
 			if (IsAPI)
 			{
-				AllowedAPIHosts.Add(Host);
+				if (URL.Host == "raw.githubusercontent.com")
```

Comment:
```suggestion
				if (URL.Host == "raw.githubusercontent.com")
```
move the string to a constant